.github/workflows/testing.yml

```diff
@@ -0,0 +1,50 @@
+# Builds wheels and libraries used to run tests, uploads them, then runs the standard tests.
+name: Build and run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: "12.4"
+      - name: Workaround ARM64 issues
+        # https://github.com/actions/virtual-environments/issues/2557
+        run: |
+          sudo rm -Rf /Library/Developer/CommandLineTools/SDKs/*
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Build libs and wheels
+        # Use a venv from multibuild for this step.
+        run: |
+          source multibuild/osx_utils.sh
+          get_macpython_environment $MB_PYTHON_VERSION venv
+          make
+        env:
+          MB_PYTHON_VERSION: "3.9"
+          MB_PYTHON_OSX_VER: "10.9"
+      - name: Upload test data
```

Comment:
Should we upload after the tests?

Reply:
The upload is so the test data can be viewed.  I might want to look at that it even if the tests fail.

Reply:
Fair enough ...